src/items/predefined.rs

```diff
@@ -162,6 +162,15 @@ impl PredefinedMenuItem {
         PredefinedMenuItem::new(PredefinedMenuItemType::Services, text)
     }
 
+    /// 'Bring all windows in front' menu item
+    ///
+    /// ## Platform-specific:
+    ///
+    /// - **Windows / Linux:** Unsupported.
+    pub fn front(text: Option<&str>) -> PredefinedMenuItem {
```

Comment:
I feel like `front` is a bit ambiguous and we should rename this to `bring_all_to_fron`

Reply:
Makes sense. I changed the name at dfb0f41.